api/envoy/api/v2/auth/cert.proto

```diff
@@ -336,6 +336,12 @@ message UpstreamTlsContext {
   //
   // Defaults to 1, setting this to 0 disables session resumption.
   google.protobuf.UInt32Value max_session_keys = 4;
+
+  // High risky setting! If true, client will downgrade to plain text connection.
+  // Should only be used for opportunistic encryption.
+  // .. attention::
+  // TLS renegotiation is considered insecure and shouldn't be used unless absolutely necessary.
```

Comment:
This has nothing to do with renegotiation, which is a different thing in TLS context.

Reply:
Good catch, thanks! Copy from `allow_renegotiation`. Will change it.